src/lib/spinel/radio_spinel.cpp

```diff
@@ -66,6 +66,7 @@ otRadioCaps RadioSpinel::sRadioCaps = OT_RADIO_CAPS_NONE;
 RadioSpinel::RadioSpinel(void)
     : Logger("RadioSpinel")
     , mInstance(nullptr)
+    , mCallbacks()
```

Comment:
We would not need this when calling the default constructor. I suggest not adding it.

Reply:
After I removed it, the CI reports the error `error: member ‘ot::Spinel::RadioSpinel::mCallbacks’ is used uninitialized [-Werror=uninitialized]`.